server/api/views/user.py

```diff
@@ -1,11 +1,50 @@
-from django.contrib.auth.models import User, Group
-from rest_framework import viewsets
-from api.serializers import UserSerializer, GroupSerializer
+from django.http import HttpResponseRedirect
+from django.contrib.auth.models import User
+from rest_framework import permissions, status
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from rest_framework.views import APIView
+from api.serializers import UserSerializer, GroupSerializer, UserSerializerWithToken
 
+# https://medium.com/@dakota.lillie/django-react-jwt-authentication-5015ee00ef9a
+@api_view(['GET'])
+def current_user(request):
+    """
+    Determine the current user by their token, and return their data
+    """
 
-class UserViewSet(viewsets.ModelViewSet):
+    serializer = UserSerializer(request.user)
+    return Response(serializer.data)
+
+# https://medium.com/@dakota.lillie/django-react-jwt-authentication-5015ee00ef9a
+class UserList(APIView):
     """
-    API endpoint that allows users to be viewed or edited.
+    Create a new user. It's called 'UserList' because normally we'd have a get
+    method here too, for retrieving a list of all User objects.
```

Comment:
[The official convention](https://www.python.org/dev/peps/pep-0257/) (just for the future) is that Python docstrings like this one should contain only client-facing descriptions of the functionality, and not include implementation-specific notes that consumers of your class/function won't need to know about. This is the same philosophy behind the "function contracts" we're being asked to write in C++. No need to change it this time, but keep in mind for the future.

Reply:
I just used the same code that they have in the tutorial.